[Blog Draft] GSI Tech Partner Highlight (#2716)
opensearch-project/project-website: _posts/2024-04-15-gsi-tech-blog.md

---
# List of one or more author names that map to the value of short_name in site.community_members. See the content in the _community_members collection for reference.
authors: 
    - nateboot

# Content categorization. One or more categories can be specified. 
category:
  - community

# Publish date in the format of four digit year, two digit month, two digit day, hour, minute, second, and timezone offset; e.g., 2021-04-12 01:01:01 -0700
date: 2024-04-15 01:01:01 -0700

> 

Reviewer: Please add meta: 

meta_keywords: ML model, vector store, GSI Technology, OpenSearch AI and ML

meta_description: GSI Technology uses OpenSearch as a vector store to solve advanced real world problems in the realms of AI, Aerospace, Defense and even DNA mapping.

Communications satellites are great at talking to each other, but earth observation satellites don’t always have such a luxury. They have large arrays of sensors, and collect large quantities of data. However they still have to send their data to the ground for processing. That is to say, these observational satellites don’t really compute on their own sensor data. To further complicate this issue, sending sensor data to earth has to happen in small windows of time, due to satellites only being able to transmit to ground stations in small windows in their orbit. This means that not all of their data is sure to make it. Even after all of this happens, a human still has to review the results.

The number of satellites is increasing. We have five times more satellites in space than we did ten years ago. The proportional amount of humans we have to do this processing is decreasing. Humans have become a bottleneck.

![](/assets/media/blog-images/2024-04-15-gsi-tech-blog/objects-launched-annual.jpg)

## What if...

> Suppose these types of satellites could be outfitted with a card that both stored and computed, equivalent to having both RAM and CPU, with the capability to run their sensor data through an ML model. The goal being to vectorize and pre-process sensor data. Sending a vector or change detection earthside is a much quicker operation than sending a complete set of sensor data. One of our partners, [GSI Technology](https://gsitechnology.com), might just change some of the landscape in this regard. They’ve introduced a new kind of chip into the market. The project, called “Gemini-II”, is an APU. It is an extremely power efficient processing unit that can both store and compute in its own distributed memory. You may have seen their booth at [OpenSearchCon 2023](http://www.opensearch.org/events/opensearchcon/2023/north-america/index.html).

Reviewer 2: ```suggestion
Suppose that a satellite could be outfitted with a card that both stored and computed its own distributed memory (equivalent to having both RAM and CPU) with the capability to run sensor data through an ML model, the goal being to vectorize and pre-process sensor data. Sending a vector or change detection Earthside is a much quicker operation than sending a complete set of sensor data. One of our partners, [GSI Technology](https://gsitechnology.com), might just change the landscape in this regard. They’ve introduced a new kind of chip to the market. The project, called “Gemini-II”, is an APU. It is an extremely power-efficient processing unit that can both store and compute its own distributed memory. You may have seen their booth at [OpenSearchCon 2023](http://www.opensearch.org/events/opensearchcon/2023/north-america/index.html).
```

> Suppose these types of satellites could be outfitted with a card that both stored and computed, equivalent to having both RAM and CPU, with the capability to run their sensor data through an ML model. The goal being to vectorize and pre-process sensor data. Sending a vector or change detection earthside is a much quicker operation than sending a complete set of sensor data. One of our partners, [GSI Technology](https://gsitechnology.com), might just change some of the landscape in this regard. They’ve introduced a new kind of chip into the market. The project, called “Gemini-II”, is an APU. It is an extremely power efficient processing unit that can both store and compute in its own distributed memory. You may have seen their booth at [OpenSearchCon 2023](http://www.opensearch.org/events/opensearchcon/2023/north-america/index.html).

Reviewer 2: APU should be defined. "auxiliary power unit (APU)"?

> The APU technology family is a Compute-in-memory (CiM) architecture seeking to eliminate computation time wasted on data movement by closing the traditional gap between processors (millions in GSI's case) and storage elements, which is highly beneficial for data-intensive workloads. Computation occurs where the data lives - in the storage elements, avoiding traditional Processor/Memory bottlenecks!

Reviewer 2: ```suggestion
The APU technology family is a Compute-in-memory (CiM) architecture that eliminates computation time wasted on data movement by closing the traditional gap between processors (millions in GSI's case) and storage elements, which is highly beneficial for data-intensive workloads. Computation occurs where the data lives---in the storage elements---avoiding traditional processor/memory bottlenecks.
```

## Several applications

> Many institutions often use things like SAR (synthetic aperture radar) to monitor landscapes to check for things like the progress of deforestation, forest fires, as well as the erosion of coastal areas. Even corporations use this tech to count the number of cars in their stores’ parking lots to help forecast business decisions. We can even use this tech to predict and detect ships adrift at sea for the purposes of rescue. The same advantage is available here. Instead of sending entire collections of satellite data, we now have the potential to send results that represent those changes instead.

Reviewer 2: ```suggestion
Many organizations use technology like synthetic aperture radar (SAR) to monitor landscapes to check for things like deforestation progress, forest fires, or erosion of coastal areas. Even corporations use this technology to count the number of cars in their stores’ parking lots to help forecast business decisions. We can even use it to predict and detect ships adrift at sea for the purposes of rescue. The same advantage is available here. Instead of sending entire collections of satellite data, we now have the potential to send results that represent those changes instead.
```

> If you’re curious about what GSI Technology works on you should definitely head over to their [website](https://gsitechnology.com). They’re applying AI and ML to a variety of modern problems. There are many potential applications that are up to their imagination - aerospace, genomics, computer vision, cyber security for space, vector search in space, data centers in space for the purposes of computing large volumes of data and more.

Reviewer 2: ```suggestion
If you want to learn more about GSI Technology solutions, you should check out [their website](https://gsitechnology.com). They’re applying AI and ML to a variety of modern problems. There are many potential applications, including aerospace, genomics, computer vision, cybersecurity in space, vector search in space, data centers in space for the purposes of computing large volumes of data, and more.
```

> ## Parting food for thought

Reviewer 2: ```suggestion
## Parting thoughts
```

> ## Parting food for thought

Reviewer 2: Try not to use idiomatic expressions, as they will likely not be understood by portions of our audience.

> Satellites and space sure sound cool, but there may be many other places in which the observability of our own Earth can be improved. Not only on a cosmic scale, but any place in which we collect large amounts of data. Imagine a microscope with a bit of computational memory, equivalent to RAM and a CPU that could supplement the years and years of experience doctors and scientists have accumulated. Not replace, but supplement. Imagine an MRI machine equipped with a model to recognize brain scan anomalies. As a long time sufferer of epilepsy, a condition for which only three out of ten cases ever have a root cause determined, I sure wouldn’t mind donating the vector of my brain scans to science. With an open source license, of course.

Reviewer 2: ```suggestion
Satellites and space sure sound cool, but there may be many other places in which the observability of our own planet can be improved. This may be true not only on a cosmic scale but for any application in which you collect large amounts of data. Imagine a microscope with a bit of computational memory, equivalent to RAM and a CPU, that could supplement the years and years of experience that doctors and scientists have accumulated (not replace, but supplement). Imagine an MRI machine equipped with a model that recognizes brain scan anomalies. As a longtime sufferer of epilepsy, a condition for which only 3 out of 10 cases ever have a root cause determined, I sure wouldn’t mind donating the vector of my brain scans to science---with an open-source license, of course.
```
